src/Symfony/Component/Translation/Extractor/PhpExtractor.php

```diff
@@ -196,9 +196,10 @@ private function getValue(\Iterator $tokenIterator)
      * Extracts trans message from PHP tokens.
      *
      * @param array            $tokens
+     * @param string           $filename
      * @param MessageCatalogue $catalog
      */
-    protected function parseTokens($tokens, MessageCatalogue $catalog)
+    protected function parseTokens($tokens, $filename, MessageCatalogue $catalog)
```

Comment:
That's a BC - it's not allowed.
I'd suggest adding the filename as last argument, but commented for now.
See how it's done e.g. on https://github.com/symfony/symfony/blob/master/src/Symfony/Component/DomCrawler/Crawler.php#L502

Reply:
Sorry, for some reason I thought `parseTokens` was `private`. Then it would not be a BC break, right? Clever workaround! I'll put it in.